recipes/varscan/2.3.7/varscan.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
```

Comment:
Mh I though about this one: https://github.com/bioconda/bioconda-recipes/tree/master/recipes/peptide-shaker